client/verta/verta/operations/monitoring/monitored_entity.py

```diff
@@ -100,6 +148,20 @@ def _update(self, msg, response_proto, endpoint, method):
 
 
     def delete(self):
+        """Delete this monitored entity.
+
+        Instructs Verta's services to delete this monitored entity.
+
+        Returns
+        -------
+        bool
+            True if the delete was successful.
+
+        Raises
+        ------
+        Error
+            Raises an error if the delete failed for any reason.
```

Comment:
I personally usually forgo the `Raises` section unless there's a particular exception case that's unintuitive/surprising. I'm open to cataloguing them more thoroughly if that seems to be a user-friendlier approach.

Reply:
Fair enough.

Reply:
Actually, it might make sense for `delete()`. The function returns `True` if the delete succeed, but raises an exception otherwise as opposed to returning `False`.